ihatemoney/templates/download_mobile_app.html

```diff
@@ -0,0 +1,23 @@
+{% extends "layout.html" %}
+
+{% block css %}
+<link rel=stylesheet type=text/css href="{{ url_for("static", filename='css/download_mobile_app.css') }}">
+{% endblock %}
+
+{% block body %}
+<main class="py-5 text-center">
+
+    <div class="mx-auto text-center" style="background-color: black;">
+        <h1>Download Mobile Application</h1>
+    </div>
+    <h1>Get it on</h1>
+    <a href="https://play.google.com/store/apps/details?id=net.eneiluj.moneybuster&hl=en_US">
+        <img src="{{ url_for("static", filename='images/google-play.png') }}" class="get-it-from"/>
+    </a>
+
+    <a href="https://f-droid.org/en/packages/net.eneiluj.moneybuster/">
+    <img src="{{ url_for("static", filename='images/f-droid.svg') }}" class="get-it-from" />
+    </a>
+
+</main>
```

Comment:
```suggestion
<main class="row py-5 justify-content-center text-center">

    <div class=" col-10 text-center bg-dark text-light">
        <h1 class="my-2">{{_("Download Mobile Application")}}</h1>
    </div>
    <h3 class="col-12 my-2">{{_("Get it on")}}</h1>
    <div class="col-12">
        <a target="_blank" rel="noopener" href="https://play.google.com/store/apps/details?id=net.eneiluj.moneybuster&hl=en_US" class="text-decoration-none">
            <img src="{{ url_for("static", filename='images/google-play.png') }}" class="get-it-from"/>
        </a>
    
        <a target="_blank" rel="noopener" href="https://f-droid.org/en/packages/net.eneiluj.moneybuster/" class="text-decoration-none">
            <img src="{{ url_for("static", filename='images/f-droid.svg') }}" class="get-it-from" />
        </a>
    </div>

</main>
```
There are a lot of things here, but mainly:
- Use bootstrap grid system
- Use bootstrap colors and text decoration
- Use bootstrap spacing
- Internationalization

I personally change from `h1` to `h3` to have a smaller `Get it on`, but this is opened to discussion.